cspatients/tests.py

```diff
@@ -1,18 +1,108 @@
 from django.test import Client
 from django.test import TestCase
 from django.contrib.auth.models import User
+from django.http import QueryDict
 
 from .models import Patient, PatientEntry
 from .serializers import PatientSerializer, PatientEntrySerializer
-from .util import (view_all_context, save_model, save_model_changes)
+from .util import (view_all_context, save_model, save_model_changes,
+                   get_rp_dict)
 
+import urllib.parse as parse
 # View Tests
 
 
 def delete_whole_table():
     PatientEntry.objects.all().delete()
```

Comment:
I know this isn't part of the PR, but I think you might need to review the use of this function. To an outsider, I think this function obscures what's happening, rather than improving readability or functionality of the code.

Usually, when something is a single line, it might not be worth creating a function for it. Functions work better as tools for encapsulating a group of behaviors or actions.